pact-tests/pact-tests/db.repl

```diff
@@ -150,26 +150,26 @@
 
 ;; test nested commits
 
-(begin-tx)
-(env-enable-repl-natives true)
-(module nested-tx G
-  (defcap G () true)
-  (defschema s x:integer)
-  (deftable t:{s})
-  (defun test-nested-tx ()
-    (begin-tx)
-    (insert t "a" { 'x: 1 })
-    (commit-tx)
-    (begin-tx)
-    (insert t "b" { 'x: 2 })
-    (rollback-tx)
-    (expect "2nd insert rolled back" ["a"]
-            (keys t))))
-
-(create-table t)
-(commit-tx)
-
-(nested-tx.test-nested-tx)
+;  (begin-tx)
```

Comment:
This test needs to be rewritten, but basically, `defun test-nested-tx` is an ugly corner case of the language we allow in the repl, but now we cannot allow it to preserve free variables across `begin-tx` boundaries. This test needs minor rewriting.